userpass

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+#
+# Exports the mm2 userpass.
+#
+# @author lightspeed393
+#
+scriptpath="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+source $scriptpath/main
+
+export userpass="$userpass"
```

Comment:
Possibly not I was originally trying not to use it, but mpm was not reading the userpass exported from `./start`.

Reply:
I think it's because normally someone would run `start` in one terminal and then other commands in another, start's `export` would not set the env var in that other terminal. hmm..

Reply:
Yes that makes sense. I was using multiple terminals when testing.

Reply:
I'm wondering if it should just be added to `mpm_run` and `mpm_start` instead of adding an extra file/command

Just add to the top of those
```
if (( mpm_enable > 0 )); then
    export userpass=${userpass}
fi
```

Reply:
Good call thanks. Done and tested.

Reply:
Nice